pkg/cluster/internal/providers/docker/network_test.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package docker
+
+import (
+	"testing"
+)
+
+func Test_generateULASubnetFromName(t *testing.T) {
+	t.Parallel()
+	cases := []struct {
+		name   string
+		subnet string
+	}{
+		{
+			name:   "kind",
+			subnet: "fc00:5ae0:f40:c847::/64",
+		},
+		{
+			name:   "kind2",
+			subnet: "fc00:cadf:7baf:98ea::/64",
+		},
+		{
+			name:   "kin",
+			subnet: "fc00:f804:3187:7254::/64",
+		},
+		{
+			name:   "mysupernetwork",
+			subnet: "fc00:1dd9:697c:314e::/64",
+		},
+	}
+	for _, tc := range cases {
+		tc := tc
```

Comment:
is the local variable needed here?

Reply:
https://github.com/golang/go/wiki/CommonMistakes#using-reference-to-loop-iterator-variable

Reply:
i don't think it's needed because the iteration happens over a slice of values.

Reply:
see the second example under the link above, there's a single loop variable so you need to capture it. normally I leave a comment something like:
```go
tc := tc  // capture variable
```

Reply:
ok, right this is the go routine in a loop gotcha.
it's required when using t.Parallel() in a sub-test.

Reply:
it's also more generally true when capturing either when passing a reference explicitly or when doing a closure.

you can explicitly capture instead for the closure case but this assignment / scoping hack is convenient for all cases :P